conda/recipes/libcucim/build.sh

```diff
@@ -4,7 +4,6 @@ CUCIM_BUILD_TYPE=${CUCIM_BUILD_TYPE:-release}
 
 echo "CC          : ${CC}"
 echo "CXX         : ${CXX}"
-echo "CUDA        : ${CUDA}"
```

Comment:
Dropping this line as it is unset & unused based on our prior discussion. This resolves issue ( https://github.com/rapidsai/cucim/issues/582 )